### PR TITLE
Update to polkadot-v0.9.22

### PR DESCRIPTION
**Pull request by brunopgalvao**
Part of https://github.com/substrate-developer-hub/substrate-docs/pull/1045.

### CODE REVIEW

**Comment by NukeManDan** [comment]
does this incude https://github.com/paritytech/cumulus/pull/1287#event-6663399069? And do all templates do as well? Should be updated and tags moved as needed on those templates before we merge this in @brunopgalvao

**Comment by brunopgalvao** [comment]
AFAIU, changes made in https://github.com/paritytech/cumulus/pull/1287 are outside of the parachain-template therefore no additional changes need to be made to https://github.com/substrate-developer-hub/substrate-parachain-template/pull/111

If that is the case, then https://github.com/substrate-developer-hub/substrate-parachain-template/pull/111 is ready for review. Once it's merged, tags can be created.

**Comment by NukeManDan** [comment]
Cargo sets the commit it used and doesn't change unless explicitly said to in the lock file. Thus an update is needed. see https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html for more details.

File: env.development [hunk 1]
```diff
@@ -0,0 +1,4 @@
+GATSBY_WEBSITE_URL=http://localhost:8100
```
Comment (NukeManDan) [inline env.development:1]: this shouldn't be here, should be a dot file too, right? ;) That way it's caught by `.gitignore`

File: static/assets/tutorials/cumulus/node-start-local-testing.sh [hunk 2]
```diff
@@ -18,7 +18,7 @@ rm -rf /tmp/relay /tmp/parachain
 # Relay Validators with 2 validator chain spec
 ./target/release/polkadot --alice --validator --base-path /tmp/relay/alice --chain ../docs/static/assets/tutorials/cumulus/chain-specs/rococo-custom-2-raw.json --port 30333 --ws-port 9944
 
-./target/release/polkadot --bob --validator --base-path /tmp/relay/bob --chain ../docs/static/assets/tutorials/cumulus/chain-specs/rococo-custom-2-raw.json --port 30334 --ws-port 9945
+./target/release/polkadot --bob --validator --base-path /tmp/relay/bob --chain ../docs/static/assets/tutorials/cumulus/chain-specs/rococo-custom-2-raw.json --bootnodes /ip4/127.0.0.1/tcp/30333/p2p/<bootnode_id_here> --port 30334 --ws-port 9945
```
Comment (NukeManDan) [inline static/assets/tutorials/cumulus/node-start-local-testing.sh:21]: this was never needed before. Why is it now?

**Review by NukeManDan — CHANGES_REQUESTED** [review]

File: static/assets/tutorials/cumulus/node-start-local-testing.sh [hunk 3]
```diff
@@ -45,7 +45,7 @@ rm -rf /tmp/relay /tmp/parachain
 
 
 # Start Parachain collator, assumes ../docs/static/assets/tutorials/cumulus/chain-specs/rococo-custom-2-raw.json was updated in docs (working dir for this repo is set as `docs` below, beside the working dirs for polkadot and for the parachain template).
-./target/release/parachain-collator --alice --collator --force-authoring --chain rococo-local-parachain-2000-raw.json --base-path /tmp/parachain/alice --port 40333 --ws-port 8844 -- --execution wasm --chain ../docs/static/assets/tutorials/cumulus/chain-specs/rococo-custom-2-raw.json --port 30343 --ws-port 9977
+./target/release/parachain-collator --alice --collator --force-authoring --chain rococo-local-parachain-2000-raw.json --base-path /tmp/parachain/alice --port 40333 --ws-port 8844 -- --execution wasm --chain ../docs/static/assets/tutorials/cumulus/chain-specs/rococo-custom-2-raw.json --bootnodes /ip4/127.0.0.1/tcp/30333/p2p/<bootnode_id_here> --port 30343 --ws-port 9977
```
Comment (NukeManDan) [inline static/assets/tutorials/cumulus/node-start-local-testing.sh:48]: same

File: env.development (hunk 1 above)
Comment (brunopgalvao) [reply env.development]: removed.

File: static/assets/tutorials/cumulus/node-start-local-testing.sh (hunk 2 above)
Comment (brunopgalvao) [reply static/assets/tutorials/cumulus/node-start-local-testing.sh]: you are correct, it auto-detects - not needed.

**Comment by brunopgalvao** [comment]
I resolved merge conflicts [here](https://github.com/substrate-developer-hub/substrate-docs/pull/1053/commits/99f491561b8f89722867391cbc4e5aa808391569).

**Review by NukeManDan — APPROVED** [review]
so chainspecs only :ok:

**Pull request merged**